risc0/build/src/lib.rs

```diff
@@ -225,29 +216,20 @@ where
 pub fn embed_methods() {
     let pkg = current_package();
 
-    let out_dir_env = env::var_os("OUT_DIR").unwrap();
-    let out_dir = Path::new(&out_dir_env);
-    let guest_target_dir = out_dir.join("riscv-guest");
+    let out_path = env::var_os("OUT_DIR").unwrap();
```

Comment:
I think `out_dir_env` is slightly better name since the thing returned from `env::var_os` isn't actually a path but a string.